exec/java-exec/src/test/java/org/apache/drill/TestExampleQueries.java

```diff
@@ -1195,4 +1195,25 @@ public void testDateImplicitCasting() throws Exception {
         .run();
   }
 
-}
\ No newline at end of file
+  @Test // DRILL-4682
+  public void testColumnNamesWithSchemaName() throws Exception {
+    test("SELECT cp.`employee.json`.department_id FROM cp.`employee.json` ORDER BY cp.`employee.json`.department_id");
```

Comment:
first comment. Please use TestBuilder to verify some query result. Using test() only verifies query is completed.